playbook/app/pb_kits/playbook/pb_caption/caption.rb

```diff
@@ -15,8 +15,8 @@ class Caption < Playbook::KitBase
                      default: nil
 
       prop :color, type: Playbook::Props::Enum,
-                   values: [nil, "light", "default", "link"],
-                   default: nil
+                   values: %w[default light lighter success error link],
+                   default: "default"
```

Comment:
We want to keep default color to be nil here. It's tricky because default is actually a color name set to a value(see line 25 in sass file).